cypress/integration/plugins/dashboards-flow-framework/create_workflow_spec.js

```diff
@@ -132,20 +132,8 @@ describe('Creating Workflows Using Various Methods', () => {
       .should('be.visible')
       .click();
     cy.fixture(FF_FIXTURE_BASE_PATH + 'semantic_search/ingest_response').then(
```

Comment:
Tyler, In the fixture please remove file`semantic_search/ingest_response`. As it is no longer used.

Reply:
Prefer to leave actually, so we're mocking correctly. Also, it actually is still required, as once ingest responses are returned, we have logic to to open up and auto-switch tabs to ingest or search, so the visibility check is still valid.

In the future, we may add some more flexible logic to parse out the responses and ensure there are visible parts in them as well.